README.Rmd

```diff
@@ -26,75 +26,91 @@ knitr::opts_chunk$set(
 [![Coverage Status](https://img.shields.io/codecov/c/github/r-lib/pak/master.svg)](https://codecov.io/github/r-lib/pak?branch=master)
 <!-- badges: end -->
 
-pak installs R packages from various sources.
+pak installs R packages from various sources. It is a replacement for
```

Comment:
I think you could spell out it here: pak installs R packages from CRAN, Bioconductor, GitHub, and ...